Makefile

```
@@ -153,9 +153,9 @@ helm-install-worker:
 	--set useCredSecret=false
 
 make helm-uninstall:
-	helm uninstall worker
-	helm uninstall root
-	helm uninstall recruiter
+	helm uninstall worker || :
+	helm uninstall root || :
+	helm uninstall recruiter || :
```

https://www.gnu.org/software/make/manual/html_node/Errors.html
```suggestion
	-helm uninstall worker
	-helm uninstall root
	-helm uninstall recruiter
```